src/Agoda.Analyzers/RuleContent/AG0039MethodLineLengthAnalyzer.html

```diff
@@ -0,0 +1,69 @@
+﻿<p>
+    Long methods should be considered for refactoring
+</p>
+<ul>
+    <li>
+        This rule is to help people identify potentially long methods, so they can consider refactoring them.
+    </li>
+    <li>
+        The rule does not count lines of whitespace in the limit of 40 lines.
```

Comment:
Still refers to constant 40

Reply:
im unable to programmatically set this one, so need to ahve in two places. unless you know how i can do it